### ISSUE TITLE
@unlink

### ISSUE THREAD

**Issue by Tarendai** [humanmade/WordPress-Importer#25]
The WXR Importer class contains a number of `@unlink` calls, these should be wrapped in checks to see if the file is readable/writable/exists, and the `@` operator removed

**Comment by rmccue** [comment]
@Tarendai Good catch, these should use `error_reporting` correctly instead. (I've had this problem in the past when a stream wrapper like S3 throws an exception, which silently gets eaten.)
